vmms/distDocker.py

```diff
@@ -68,12 +67,8 @@ class DistDocker:
 
     _SSH_FLAGS = ["-q", "-o", "BatchMode=yes" ]
     _SSH_AUTH_FLAGS = [ "-i", os.path.join(os.path.dirname(__file__), "id_rsa"),
-              "-o", "StrictHostKeyChecking=no",
-              "-o", "GSSAPIAuthentication=no"]
-    _SSH_MASTER_FLAGS = ["-o", "ControlMaster=yes",
```

Comment:
Why are these removed? Don't we want to use multiplexing? Maybe a better flag may be `ControlMaster=auto` which will use the master connection if one exists, and create one otherwise.